membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -44,7 +44,8 @@ object AccountDetails {
           "trialLength" -> paymentDetails.remainingTrialLength,
           "plan" -> Json.obj(
             "name" -> paymentDetails.plan.name,
-            "amount" -> paymentDetails.plan.amount,
+            "amount" -> paymentDetails.plan.amount.amount * 100,
+            "currency" -> paymentDetails.plan.amount.currency.glyph,
```

Comment:
It would be probably better to put the currency 3-letters code here instead of the glyph.